src/views/product-tutorials-view/index.tsx

```diff
@@ -31,20 +41,24 @@ function ProductTutorialsView({
         />
       }
     >
-      <h1 id={layoutProps.headings[0].slug}>{product.name} Tutorials</h1>
+      <PageHeading />
       <ProductViewContent
         blocks={blocks}
         inlineCollections={inlineCollections}
         inlineTutorials={inlineTutorials}
       />
-      {showProductSitemap ? (
-        <>
-          <h2 id={layoutProps.headings[layoutProps.headings.length - 1].slug}>
-            All tutorials
-          </h2>
-          <ProductTutorialsSitemap collections={allCollections} />
-        </>
-      ) : null}
+      {showProductSitemap
```

Comment:
The inline anonymous function here is kinda gritty. Open to moving this; my hot take was we could live with the awkwardness for now and move this into the `sitemap` component itself in the subsequent PR where we implement `sitemap` to spec.

Reply:
Agreed, creating its own component when you go to style makes sense.